scripts/hud/cgaz.js

```diff
@@ -250,6 +250,12 @@ class Cgaz {
 		const accel = lastMoveData.acceleration;
 		const maxAccel = accel * maxSpeed * tickInterval;
 
+		if (lastMoveData.hasteEndTime < 0 || lastMoveData.hasteEndTime > MomentumMovementAPI.GetCurrentTime()) {
+			this.snapAccel = 3.328; // (max speed) * (air accel = 1) * (tick interval) * (haste factor)
+		} else {
+			this.snapAccel = 2.56;
```

Comment:
Is there a way to change these to constants instead of magic numbers?

Reply:
yep, good look. I've defined these as constants now